src/pages/Chat.vue

```diff
@@ -32,7 +32,7 @@
             </q-popup-edit>
             <!-- Display the role of the user or the AI -->
             <q-item-label class="text-semibold q-mb-md">
-              {{ message.role }}
+              {{ message.role.replace('assistant', 'Libertai Assistant (default)') }}
```

Comment:
why ? This isn't very generic and clean, we should remove it and fix the underlying problem (if any) in another way

Reply:
should be fixed adding a role field instead of label